client/src/components/InputTextField/InputTextField.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "./InputTextField.css";
+import "../../App.css";
+/**
+ * Created by Tyler
+ *
+ * Props will define the type of input field
+ * Input fields include email, password, name, confirm password, city of residence, what is your age?, what is your dream destination, First name,   last name, search my directory, search trips.
+ *
+ * I think we'll need to implement a function on these to retrieve the data typed in when pressing a submit button. I wasn't sure how to do this with just the standalone components in storybook.
+ */
```

Comment:
This documentation is great! We're going to figure out a way to provide this kind of component documentation so you don't have to do this